js/_website/src/lib/components/DemosLite.svelte

```diff
@@ -82,22 +84,18 @@
 	$: code = demos.find((demo) => demo.name === current_selection)?.code || "";
 	$: requirements =
 		demos.find((demo) => demo.name === current_selection)?.requirements || [];
+	$: requirementsStr = JSON.stringify(requirements); // Use the stringified version to trigger reactivity only when the array values actually change, while the `requirements` object's identity always changes.
 
 	$: if (mounted) {
-		if (timeout) {
-			clearTimeout(timeout);
-		}
-		timeout = setTimeout(() => {
-			update(code, requirements);
-		}, 1000);
+		debounced_run_code && debounced_run_code(code);
```

Comment:
Refactoring: when the code is changed, call only `run_code` without `install()`.